### PR TITLE
ToInteger decorator

### PR DESCRIPTION
**Pull request by Velmisov**
`ToInteger` decorator is equal to `Transform` with `parseInt` convert

### CODE REVIEW

File: src/decorators.ts [hunk 1]
```diff
@@ -36,6 +36,15 @@ export function ToBoolean() {
     });
 }
 
+/**
+ * Cast value to integer type
+ */
+export function ToInteger(radix?: number) {
```
Comment (Goodluckhf) [inline src/decorators.ts]: I think we can provide default radix as 10

**Review by Goodluckhf — CHANGES_REQUESTED** [review]

File: src/decorators.ts (hunk 1 above)
Comment (Velmisov) [reply src/decorators.ts]: It is. Just the same as `parseInt` 
image

Comment (Goodluckhf) [reply src/decorators.ts]: I mean to set explicitly default value as 10. Because almost in all cases we use 10 radix

Comment (Velmisov) [reply src/decorators.ts]: Check

**Review by Goodluckhf — APPROVED** [review]

**Pull request merged**